icu4c/source/common/uniset_props.cpp

```diff
@@ -657,16 +657,16 @@ void UnicodeSet::applyPattern(RuleCharacterIterator& chars,
 
 namespace {
 
-static UBool numericValueFilter(UChar32 ch, void* context) {
+UBool numericValueFilter(UChar32 ch, void* context) {
```

Comment:
> We're removing the `static` because these functions were already in the anonymous namespace, I take it?

yes

> Also, should these still be `UBool` if they're internal?

I am not motivated right now to change that in existing code. And implementing an existing interface should count as existing code.

Reply:
>I am not motivated right now to change that in existing code. And implementing an existing interface should count as existing code.

Fair enough.